recipe/meta.yaml

```
@@ -110,6 +110,7 @@ outputs:
     build:
       run_exports:
         - {{ pin_subpackage("cuda-cudart", max_pin="x") }}
+        - {{ pin_subpackage("cuda-cudart_" + target_platform, max_pin="x") }}
```

Think we want to restrict this to Windows

Linux doesn't have this issue

```suggestion
        - {{ pin_subpackage("cuda-cudart_" + target_platform, max_pin="x") }}  # [win]
```